config/cfg_mgmt.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bgp": {
+        "container": "bgp",
+        "files": [
+            "/etc/frr/bfdd.conf",
+            "/etc/frr/bgpd.conf",
+            "/etc/frr/ospfd.conf",
+            "/etc/frr/pimd.conf",
+            "/etc/frr/staticd.conf",
+            "/etc/frr/zebra.conf"
+        ]
+    }
+}
```

Comment:
This might be opening a can of worms, but how about other `/etc` files like `/etc/passwd` `/etc/groups` and `/etc/shadow`?

Reply:
Those files are already handled by the existing upgrade scripts. Depending on the need, we can add additional files and/or folders to this list.